packages/xo-server/src/api/vm.js

```diff
@@ -756,6 +757,30 @@ export { convertToTemplate as convert }
 
 // -------------------------------------------------------------------
 
+export async function cloneToTemplate({ vm }) {
```

Comment:
Why did you choose to clone instead of converting directly as we do today for the whole VM?

Reply:
Is not possible to convert a snapshot to a template. The xapi is giving an error `is_a_snapshot`.

Reply:
Also, it's not possible to  change `is_a_snapshot` of an existing VM.

In that case it makes sense to use this behavior.